frontend/src/components/PageFooter.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
+        <footer>Copyright © Code for Poznan 2019</footer>
+    </div>
+</template>
+
+<script>
+export default {
+    
+}
+</script>
+
+<style lang="scss" scoped>
+@import "./../main.scss";
+footer {
+    color: $white;
+    background-color: $blue-nav-footer;
+    padding: 25px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
I'm not sure if this line(align-items) is needed, it doesn't seem to do anything (correct me if I'm wrong).

Reply:
It aligns items vertically, but in this case you're right - it's not needed.